react/src/components/molecules/SocialLinks/index.js

```diff
@@ -70,6 +76,8 @@ const icons = {
     width="21"
     height="17"
     viewBox="0 0 21 17"
+    focusable="no"
```

Comment:
Please use the value, **false**, instead of no.  This applies to the all changes.
`focusable="no"` still gets focus depending on the combination of the OS and IE versions.

Reply:
Cool thanks! Will do